remote/advancedcontainers/docker-options.md

```diff
@@ -0,0 +1,112 @@
+---
+Order: 16
+Area: advancedcontainers
+TOCTitle: Docker Options
+PageTitle: Remote - Containers Docker options
+ContentId:
+MetaDescription: Remote - Containers Docker options
+DateApproved: 2/4/2022
+---
+
+# Alternate ways to install Docker
```

Comment:
Maybe rearrange a bit to highlight the local installations and move Podman to the end because our support isn't quite as good as for Docker:

\# Alternate ways to install Docker
\## Windows: WSL
\## Mac: Colima
\## Linux
\## Cloud-Init VM
\## Podman
\# Other container engines

Reply:
This makes sense - I've updated the flow to follow this.